features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/postgresql/postgre-9x/sessiondata-cleanup/postgresql-session-data-cleanup.sql

```diff
@@ -344,10 +354,49 @@ auditTable :=  purgingTable||'_auditlog';
 	            GET diagnostics deleteMappingCount := ROW_COUNT;
 
 	            IF (enableLog AND logLevel IN ('DEBUG','TRACE')) THEN
-	            notice := 'BATCH DELETE FINISHED ON TABLE'||purgingSessionUserMappingTable||' WITH :'||deleteMappingCount;
+	            notice := 'BATCH DELETE FINISHED ON TABLE '||purgingSessionUserMappingTable||' WITH :'||deleteMappingCount;
```

Comment:
Do we need this change?

Reply:
This is only a formatting fix to add a missing space to the log message